client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1712,9 +1712,14 @@ Undocumented.prototype.sitePurchases = function( siteId, fn ) {
 	this.wpcom.req.get( { path: '/sites/' + siteId + '/purchases' }, fn );
 };
 
-Undocumented.prototype.googleAppsListAll = function( domainName, fn ) {
+Undocumented.prototype.googleAppsFilterByDomain = function( domainName, fn ) {
 	debug( '/domains/:domainName/google-apps' );
-	this.wpcom.req.get( { path: '/domains/' + domainName + '/google-apps' }, fn );
+	return this.wpcom.req.get( { path: '/domains/' + domainName + '/google-apps' }, fn );
+};
+
+Undocumented.prototype.googleAppsFilterBySiteId = function( siteId, fn ) {
+	debug( '/sites/:siteId/google-apps/site' );
```

Comment:
It should be `/sites/:siteId/google-apps` (without `/site` at the end).
Stupid nitpick, but someone might get confused by this print and search for a non-existing endpoint.